canvas_modules/common-canvas/src/palette/palette.scss

```diff
@@ -95,9 +93,9 @@ $palette-border-color: $ui-03;
 		display: flex;
 		flex-direction: column;
 		align-items: center;
-		/* Prevent elements from being selectable */
-		/* This stops it being dragged incorrectly.*/
```

Comment:
Requesting for a third time that you put back these comments and position them above the `user-select: none;`  line.

Reply:
I did put the comment back